src/main/java/nl/knaw/dans/ingest/core/service/ZipFileHandler.java

```diff
@@ -39,6 +39,18 @@ public class ZipFileHandler {
         "application/fits-gzipped"
     );
 
+    public Path zipOriginalMetadata(Path... xml) throws IOException {
+
+        var tempFilePath = tempDir.resolve(String.format("original-metadata-%s.zip", UUID.randomUUID()));
```

Comment:
Why the UUID?

Reply:
Same strategy as for the wrapped zip file. `fileMeta.setLabel` does not use that UUID.